### ISSUE TITLE
Option to adjust HD and BAB of some classes to be more like 5ed

### ISSUE THREAD

**Issue by julianosott** [RedChimera/IWD2EE#105]
Hi! It's a suggestion. I can't log into Beamdog's forum, so I'm posting it here.

TLDR: An option to increase the HD for Wizard/Sorcerer to d6 (from d4), and BAB to 75% (from 50%). Also increase the HD of Rogues/Bards to d8 (from d6). Could choose those options individually, none or all of them.

Justification: To be more in line with D&D 5ed. The HD would be the same of 5ed, and in 5ed every class have the same Proficiency progression, only the warrior classes (Fighter, Paladin, Barbarian and Ranger) get extra attacks, so they would be the only ones to get 100% BAB. Theoretically, in 5ed the Wizard and Sorcerer have the same "BAB" (Proficiency) of the Cleric and Rogues (no extra attacks).

I understand that IWD2 is 3.5ed, not 5ed, but the REAL justification is to make a Battlemage (front-line mage) more viable and fun to play. They are too squish and miss too much. Maybe some spells must be revised, because increasing the HD and BAB of wizard and sorcerer would make Tenser's Transformation too powerful, for example. Or those spells could stay as they are. I can create this mod myself, but it would be nice to have it integrated into this mod.

**Comment by OlvynChuru** [comment]
You can absolutely make a front-line mage in IWD2EE already. Just play a multi-classed fighter/wizard.

If you'd like a single-classed wizard to be better in melee combat, you can make the mod yourself. IWD2EE is not supposed to be based on 5E (the only times IWD2EE borrows from 5E are in the case of monsters that don't appear in the 3.5E Monster Manual, such as yetis).
